x-pack/plugins/apm/public/components/app/service_overview/service_overview_charts/most_used_chart/get_lens_attributes.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  CountIndexPatternColumn,
+  TermsIndexPatternColumn,
+  PersistedIndexPatternLayer,
+  PieVisualizationState,
+  TypedLensByValueInput,
+} from '@kbn/lens-plugin/public';
+import type { Filter } from '@kbn/es-query';
+import { APM_STATIC_DATA_VIEW_ID } from '../../../../../../common/data_view_constants';
+import { MostUsedMetricTypes } from '.';
+
+const BUCKET_SIZE = 5;
+
+export function getLensAttributes({
+  metric,
+  filters,
+  kuery = '',
+}: {
+  metric: MostUsedMetricTypes;
+  filters: Filter[];
+  kuery?: string;
+}): TypedLensByValueInput['attributes'] {
+  const metricId = metric.replaceAll('.', '-');
+  const dataLayer: PersistedIndexPatternLayer = {
+    columnOrder: ['termsColumn', 'countColumn'],
+    columns: {
+      termsColumn: {
+        label: `Top ${BUCKET_SIZE} values of ${metric}`,
+        dataType: 'string',
+        operationType: 'terms',
+        scale: 'ordinal',
+        sourceField: metric,
+        isBucketed: true,
+        params: {
+          size: BUCKET_SIZE,
+          orderBy: {
+            type: 'column',
+            columnId: 'countColumn',
+          },
+          orderDirection: 'desc',
+        },
+      } as TermsIndexPatternColumn,
+      countColumn: {
+        label: 'Count of records',
```

Comment:
Do we need i18n for the labels?

Reply:
https://github.com/elastic/kibana/pull/144232/commits/8b52ee38ba3d0421eb1570a067afe8893c82573a